src/platform/plugins/shared/data_views/common/fields/field_list.test.ts

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { fieldList, IIndexPatternFieldList } from './field_list';
```

Comment:
Built on top of this PR: https://github.com/elastic/kibana/pull/224767 to test performance